ray-operator/config/samples/tpu-print-resources.py

```diff
@@ -0,0 +1,19 @@
+import ray
```

Comment:
Can you move this to ray-operator/config/samples/tpu? Also, what do you think about tpu_list_devices.py instead?

Reply:
Sounds good, done in [679e8db](https://github.com/ray-project/kuberay/pull/2445/commits/679e8db596270234dc1b578ea5ae86deb1dd4242).